stock_pack_wizard/product_ul.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2010-2015 Elico Corp (<http://www.elico-corp.com>)
+#    Alex Duan <alex.duan@elico-corp.com>
+#    Jon Chow <jon.chow@elico-corp.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.osv import fields, orm
+import openerp.addons.decimal_precision as dp
+
+
+class ProductUl(orm.Model):
+    _inherit = 'product.ul'
+
+    def _get_cbm(self, cr, uid, ids, fields, arg=None, context=None):
+        res = {}
+        for ul in self.browse(cr, uid, ids, context=context):
+            cbm = ul.high * ul.width * ul.long
+            cbm = cbm != 0 and cbm / 1000000
```

Comment:
this assumes the dimensions are in centimeters. The product information about dimensions in Odoo is in meters, and I think you should stick to this.